server/events/apply_command_runner.copy

```diff
@@ -0,0 +1,197 @@
+package events
```

Comment:
This and other `.copy` files, are they intentionally added?

Reply:
My bad. Accidentally committed some of test files. Removed now.